contributing/standards/markdown.md

```diff
@@ -89,6 +89,32 @@ def sample_func(argument1):
 
 A quick way to distinguish between the two cases when it's unclear - if this is code the student can run in a terminal, format it that way. If it's code that Exercism might run (in tests, library code the student will write, etc), default to formatting it as runnable code.
 
+## Special blocks (sometimes called admonitions)
+
+We support special types of blocks that can be added to documents to pull out commentary that doesn't fit with the main body of the text. 
+They are similar to this examples seen in the Julia docs:
+
+<img width="500" alt="Screenshot 2021-03-13 at 17 15 04" src="https://user-images.githubusercontent.com/286476/111038207-aca0bd00-841f-11eb-95fb-20a93943d3dd.png">
```

Comment:
Julia's blocks support titles, for example

```
~~~~note "This is a title"
Foo
~~~~
```

would be translated to

![grafik](https://user-images.githubusercontent.com/20866761/111039424-50916500-842e-11eb-8836-ea42e0d7992d.png)

The type of the block (represented by colour and icon) and its title are separate, though there's a default fall back if no title is specified.

Will this be supported in Exercism too?

---

Imo it would be worth it because it would allow people to easily tell apart the kind of information to expect from a box. Otherwise they'll have to read it in order to know if it's important to them.

`````
~~~~caution "Requires v1.2.3"
This feature requires Julia version v1.2.3.

In older versions, you can manually define

```julia
...
```
~~~~
`````

Reply:
> Will this be supported in Exercism too?

No, not at first at least. common-mark wouldn't support that syntax out of the box. It would need us to write a custom Markdown tokenizer, which I want to avoid in the short-term at least. But when we're out of the weeds we can think about whether it's worth the effort :)